chain/messagepool/messagepool.go

```diff
@@ -232,7 +242,13 @@ func (mp *MessagePool) Close() error {
 	return nil
 }
 
-func (mp *MessagePool) repubLocal() {
+func (mp *MessagePool) Prune() {
+	mp.pruneTrigger <- struct{}{}
+	mp.pruneTrigger <- struct{}{}
+	mp.pruneTrigger <- struct{}{}
```

Comment:
3x?

Reply:
so, its a single slot buffered channel. The first send fills the channel, the second send goes through when the pruning starts, and the third send goes through (and noops) after the pruning finishes and goes through the loop again

I should probably add a comment